src/pages/resources.tsx

```diff
@@ -12,6 +13,10 @@ export default fullPage(() => {
     return <Text>无权查看本页。</Text>;
   }
 
-  return <iframe src="https://f179b1fd0cd3453e9f34d95e95dc5f.super.site"
-    width="100%" height="700px" />;
+  return (
+      <Box h="calc(100vh - 60px)">
```

Comment:
Use {`calc(100vh - ${topbarHeight})`} instead to avoid hardcoded constants.

Reply:
addressed